public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -58,20 +59,38 @@ const LogMessage = ({ hasAnsi, entry, highlights, styles }: LogMessageProps) =>
   return <>{entry}</>;
 };
 
-const restructureLog = (line: string, prettifyLogMessage: boolean): string => {
+function extractFirstLine(input: string): string {
+  // Regular expression to match everything up to the first occurrence of '\r\n'
+  const regex = /^(.*?)(\r\n|$)/;
+  const match = input.match(regex);
+
+  if (match) {
+    return match[1];
+  } else {
+    return input;
+  }
+}
+
+const restructureLog = (isExpanded:boolean, line: string, prettifyLogMessage: boolean): string => {
```

Comment:
```suggestion
const restructureLog = (line: string, prettifyLogMessage: boolean, isExpanded:boolean): string => {
```
As a good practice, data first, and options at the end.

Reply:
Yes that way seems much better.